ContractExamples/contracts/setter/src/lib.rs

```diff
@@ -64,7 +64,7 @@ impl SetterContract {
         env.storage().instance().set(&MY_BOOL, &v);
         log!(&env, "myBool: {}", v);
         // bump the lifetime
-        env.storage().instance().extend_ttl(50, 100);
+        env.storage().instance().extend_ttl(env.storage().max_ttl() - 100, env.storage().max_ttl());
```

Comment:
Why do we subtract 100?

Reply:
+1, I had a similar question, but just assumed I was OOTL

Reply:
it's a threshold of when the TTL is to be updated. I am not exactly sure how it works, so it's a bit arbitrary.

Reply:
Makes sense. Could you add a brief comment? It looks a bit arbitrary.

Reply:
Added a comment. Actually, TTL is extended when it goes below the threshold, in this case, `max_ttl - 100`.

Reply:
Here is the fresh version. It's all modulo my understanding of how Soroban stores the storage in Stellar:

```js
    /**
     * Ordered mapping from field names to their native values (JS).
     * This mapping contains values only for the fields that have been created
     * or updated by a transaction in the past. It may happen that `fields` contains
     * fewer fields than `oldFields`, when the contract deletes some fields
     * from the storage. Also, `fields` may become empty when the storage
     * goes over TTL.
     */
```